demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -1101,6 +1101,45 @@ def is_mapping_fields_command_exist(self) -> bool:
                     return False
         return True
 
+    def are_commands_in_readme(self) -> bool:
```

Comment:
don't we need to add `are_commands_in_readme` to `is_valid_file`?

Reply:
yes, I did, Not sure why it wasnt pushed. Now it should be.